packages/office-addin-dev-settings/src/commands.ts

```diff
@@ -263,6 +251,34 @@ function parseStringCommandOption(optionValue: any): string | undefined {
   return (typeof(optionValue) === "string") ? optionValue : undefined;
 }
 
+export function parseWebView(webViewText?: string): devSettings.WebViewType | undefined {
+  const parsedWebView = webViewText ? webViewText.toLowerCase() : undefined;
+  switch (parsedWebView) {
+    case "ie":
+    case "ie11":
+    case "internet explorer":
+    case "internetexplorer":
+      return devSettings.WebViewType.IE;
+    case "edge":
+    case "spartan":
+      return devSettings.WebViewType.Edge;
+    case "edge chromium":
+    case "edgechromium":
+    case "anaheim":
+      return devSettings.WebViewType.EdgeChromium;
+    case "clear":
+      return devSettings.WebViewType.Clear;
+    case "default":
+    case "":
+      return devSettings.WebViewType.Default;
+    case null:
+    case undefined:
+      return undefined;
+    default:
+      throw new Error(`Please select a valid webview instead of '${parsedWebView}'. Options include (ie, edge, edge chromium, clear, default)`);
```

Comment:
Don't list the valid options here. Just make the message simpler. It should be phrased like other similar error messages.